synapseclient/models/user.py

```diff
@@ -0,0 +1,311 @@
+import asyncio
+from dataclasses import dataclass, field
+from typing import List, Optional, Dict, Union
+from synapseclient import Synapse
+
+from synapseclient.core.async_utils import otel_trace_method
+from synapseclient.team import (
+    UserProfile as Synapse_UserProfile,
+)
+from opentelemetry import trace, context
+from synapseclient.core.utils import run_and_attach_otel_context
+
+tracer = trace.get_tracer("synapseclient")
+
+
+@dataclass()
+class UserPreference:
+    """
+    A UserPreference represents a user preference in the system.
+
+    Attributes:
+        name: The name of the user preference.
+        concrete_type: The concrete type of the user preference.
```

Comment:
@BryanFauble are you missing concrete_type here?

Reply:
No. I just removed it from the docstring